### DISCUSSION TITLE
What to do now..?

### DISCUSSION THREAD

**Discussion by rishab-nahar** [smartcontractkit/full-blockchain-solidity-course-py#871] [General]
I have completed the course and learnt a lot doing so, Now I want to continue learning, I have looked into a few courses, and almost all of them use Java instead of Python. I want to continue working with brownie and web3.py. Is there any other source or any good way So that I can learn more.

Also, I would love to see and learn from  a few projects made with brownie and solidity.

**Comment by cromewar** [comment]
Hello @rishab-nahar yes most of the courses and resources uses JavaScript, related to this side of learning I think you already have enough tools to just start building, may be focusing on learning new technologies related to the blockchain would work, I strongly suggest you to learn Moralis.

**Reply by Verde888** [reply]
Hey @cromewar, I thought Moralis is a JS framework...

Im in a similar position to @rishab-nahar. I don't know JS, and honestly I didn't go through this course to start all over again with JS (I know JS is THE frontend language, but one just cant learn everything at the same time).

Any idea were should we look for crypto projects using Python? ... ;)

**Reply by cromewar** [reply]
I mean I don't like neither JS but let's say is a necessary pain, as is needed for a lot of stuff. But yes moralis uses JS to connect your front to them.
I don't know if there is more learning resource as good as Patrick's course, so I will recommend you to see for another projects on github (like retro cats) which are deployed using other frameworks such as truffle or hardhat, and try to reproduce them using brownie, this is an incredible way to learn.

The solidity code will remain the same anyways.

**Reply by shucks18** [reply]
@Verde888 @rishab-nahar Im in the same position as you guys any updates on your learning?
  I intend to get into JS although it'll be a pain but hey at least you'll be well versed with js and py at the end of it, I guess what @cromewar said about reproducing other frameworks with brownie makes sense ! Also we should connect on discord or something to maybe build something in the future!
